util/io/src/main/scala/sbt/IO.scala

```diff
@@ -68,18 +86,29 @@ object IO {
   }
```

Comment:
I think the semantics for the `def classLocationFile(cl: Class[_]): File` remain the same.